Userland/Applications/PixelPaint/ImageEditor.cpp

```diff
@@ -649,4 +649,12 @@ void ImageEditor::set_show_active_layer_boundary(bool show)
     update();
 }
 
+void ImageEditor::set_loaded_from_image(bool loaded_from_image)
+{
+    if (m_loaded_from_image == loaded_from_image)
```

Comment:
This is isn't really necessary... We can just always assign the value directly?

Reply:
I did this based on the boolean setter right above it. `ImageEditor::set_show_active_layer_boundary(bool show)` does the same thing.
Though I now see that that is probably because that method also needs to update the widget and needs to avoid doing that when there is no change. Changed in newest force-push.